src/main.py

```diff
@@ -2,24 +2,56 @@
 This module takes care of starting the API Server, Loading the DB and Adding the endpoints
 """
 import os
-from flask import Flask, request, jsonify, url_for
+from flask import Flask, request, jsonify, url_for, redirect
 from flask_migrate import Migrate
 from flask_swagger import swagger
 from flask_cors import CORS
+from flask_login import (
+    LoginManager,
+    current_user,
+    login_required,
+    login_user,
+    logout_user,
+)
+from oauthlib.oauth2 import WebApplicationClient
+import requests
 from utils import APIException, generate_sitemap, add_user_authentification
 from admin import setup_admin
-from models import db, User
-#from models import Person
+from models import db, User, UserOAuth
+import json
+import os
+import sqlite3
```

Comment:
Odena los imputs 
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
Done